main.tf

```diff
@@ -0,0 +1,44 @@
+module "appsync_athena_resolver" {
+  environment_variables {
+    AWS_ATHENA_S3_STAGING_DIR = "${var.athena_s3_staging_dir}"
+    AWS_ATHENA_SCHEMA_NAME    = "${var.athena_schema_name}"
+    MAX_CONCURRENT_QUERIES    = "${var.max_concurrent_queries}"
+    POLL_INTERVAL             = "${var.poll_interval}"
+  }
+
+  dead_letter_arn = "${var.dead_letter_arn}"
+  handler         = "function.handler"
+  kms_key_arn     = "${var.kms_key_arn}"
+  l3_object_key   = "quinovas/appsync-athena-resolver/appsync-athena-resolver-0.0.2.zip"
+  name            = "${var.name_prefix}-appsync-athena-resolver"
```

Comment:
take the dash off after the '}'. That way an empty name prefix doesn't generate a weird looking name